src/main/java/frc/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.subsystems.arm;
+
+public class ArmConstants {
+    public static final double PLACEHOLDER_DOUBLE = 0.0;
+    
+    public static final double ARM_POSITION_WHEN_HORIZONTAL_DEGREES = PLACEHOLDER_DOUBLE;
+
+    public static final double ARM_MOTOR_GEAR_RATIO = PLACEHOLDER_DOUBLE;
```

Comment:
This is known, it is 26:24. But I'm not sure which way it should be so wait for Basel to respond in #software